docs/index.md

```diff
@@ -14,6 +14,36 @@ layout: home
     <img src="/images/atomic-demo.gif" class="Ar(174/83) W(70%)--md" alt="Atomizer demo">
 </div>
 
+<h2>Testimonials</h2>
+
+<div id="testimonials" class="Pos(r)">
+    <ul class="Ovx(s) Ai(s) D(f)--md List(n)! Pstart(0)! Gp(2rem) H(400px)--xs Ov(h)--xs expand_H(a)">
+        {% for data in site.data.quotes %}
+            <li class="Fxb(25%) Fxs(0) Mx(0px) Mb(20px) Mb(0px)--md P(1rem) Bdrs(5px) Bgc(boxColorLight)">
+                <a href="{{ data.link }}" class="C(#000) Td(n):h" target="_blank">
```

Comment:
not the biggest fan of block level anchors but it's ok..

Reply:
Ya, was the simplest way to just make the whole thing clickable. I suppose a stretchedbox would work too

Reply:
Not a big fan of that either since SR users may choose to have links on the page read aloud to them. So a lot of text in there is kind of a no-no.
Why not using a pseudo-element in the anchor (to include the minimal relevant text) and stretch it across the whole thing?

Reply:
Ya I can looking to switching to that